.gitignore

```diff
@@ -189,3 +190,4 @@ status_log.txt
 
 # VSCode and other IDE-related ignores
 .vscode
+start_dns_agent.py
```

Comment:
`gitignore` should be universal for all members of the team, i.e., your local debug scripts don't go here :)